<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormOTO</class>
 <widget class="QWidget" name="FormOTO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OTO</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>723</width>
        <height>223</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBoxSpectrum">
           <property name="title">
            <string>spectrum</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditURLSpectrum"/>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QPushButton" name="btnSendSpectrum">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelURLSpectrum">
              <property name="text">
               <string>URL</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxInfo">
           <property name="title">
            <string>info</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="labelURLInfo">
              <property name="text">
               <string>URL</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditURLInfo"/>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QPushButton" name="btnSendInfo">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="textBrowserData"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>OTO</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textBrowserData</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>lineEditURLSpectrum</tabstop>
  <tabstop>btnSendSpectrum</tabstop>
  <tabstop>lineEditURLInfo</tabstop>
  <tabstop>btnSendInfo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
